wb-network.schema.json

```
@@ -1314,8 +1321,9 @@
             "Point-to-point address": "Адрес конечного узла",
             "For point-to-point configuration": "Для конфигурации соединения точка-точка",
             "Refers to the file in /etc/ppp/peers": "Указать имя файла конфигурации из директории /etc/ppp/peers",
-            "Managed by /etc/network/interfaces": "Управляется через /etc/network/interfaces",
-            "Managed by NetworkManager": "Управляется через NetworkManager",
+            "interfaces_description": "Настроено в /etc/network/interfaces",
+            "can_description": "Настроено в /etc/network/interfaces. Если необходимо настроить это соединение в NetworkManager'е, удалите и создайте новое",
```

В версии описания для обычной сети хочется больше настойчивости:

```suggestion
            "can_description": "Настроено в /etc/network/interfaces. Этот интерфейс нельзя настроить для автоматического переключения, пока он не управляется через NetworkManager. Для перехода на NetworkManager удалите и создайте его заново",
```

Или так (с обещаниями):

```suggestion
            "can_description": "Настроено в /etc/network/interfaces. Доступно управление через NetworkManager - это более современный сервис настройки, поддержка которого сейчас развивается в Wiren Board. Для перехода на NetworkManager удалите и создайте соединение заново",
```